test/migration/MockToken.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity >=0.8.13;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../../src/migration/IHub.sol";
+
+contract Token is ERC20 {
```

Comment:
soft fork from Hub/token V1 contracts to update to our toolstack and same compiler